File: server/bcephal-project-service/src/main/java/com/moriset/bcephal/security/domain/UserWorkspace.java
/**
 * 
 */
package com.moriset.bcephal.security.domain;

import java.util.ArrayList;
import java.util.List;

import lombok.Data;

/**
 * @author Joseph Wambo
 *
 */
@Data
public class UserWorkspace {

	private List<SubscriptionBrowserData> clients;
	private SubscriptionBrowserData defaultClient;

	private List<ProjectBrowserData> availableProjects;
	private List<ProjectBrowserData> defaultProjects;

	public UserWorkspace() {
		clients = new ArrayList<SubscriptionBrowserData>(0);
		availableProjects = new ArrayList<ProjectBrowserData>(0);
		defaultProjects = new ArrayList<ProjectBrowserData>(0);
	}

}

File: client/bcephal-client-model/Projects/UserWorkspace.cs
using Bcephal.Models.Base;
using System;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.Text;

namespace Bcephal.Models.Projects
{
    public class UserWorkspace
    {

        public ObservableCollection<BrowserData> Clients { get; set; }

        public BrowserData DefaultClient { get; set; }

        public ObservableCollection<ProjectBrowserData> AvailableProjects { get; set; }

        public ObservableCollection<ProjectBrowserData> DefaultProjects { get; set; }

    }
}
